lib/medium-admin.php

```diff
@@ -136,6 +136,17 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
+
+    if (!in_array($post->post_type, $allowed_post_types)) {
+      return;
+    }
+
+    // Use the ID of the parent post if this is a post revision
+    if ($parent_id = wp_is_post_revision($post)) {
```

Comment:
This is interesting. Under what circumstances is this true? Documentation doesn't make it clear (and it's been a *long* time since I did WP; c.f. https://wordpress.org/plugins/gengo)